src/oaklib/cli.py

```diff
@@ -3466,6 +3470,7 @@ def mappings(terms, maps_to_source, autolabel: bool, output, output_type):
                 if autolabel:
                     impl.inject_mapping_labels([mapping])
                 writer.emit(mapping)
+    writer.finish()
```

Comment:
@cmungall : Addition of this line creates the error you see in gh actions

Reply:
The output file is blank and hence the file parsed back contains nothing.

Reply:
can you go ahead and make the change?

Reply:
oh I see this is on a fork... we can get @kimrutherford set up with permissions to make branches off origin if that fits out flow better